app/views/chatrooms/show.html.erb

```diff
@@ -1,8 +1,9 @@
+<% other_user = @chatroom.other_user(current_user) %>
+<%= render 'shared/back_bar', text: "#{other_user.first_name} #{other_user.last_name}", user: other_user, link: my_messages_path %>
```

Comment:
The partial by itself will always show just the arrow, but it can have 3 more arguments:
- link to change the link for the back button. it's :back (go to previous page) by default.
- user to pass a user in order to show the user photo and link it to the user profile.
- text to pass a title for the back bar.
Most of the back buttons will only have the arrow and maybe the title. This one for the chatroom was the most complex.